orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/servergroup/CreateServerGroupStage.groovy

```diff
@@ -121,6 +121,11 @@ class CreateServerGroupStage extends AbstractDeployStrategyStage {
       additionalRollbackContext.disableOnly = true
     }
 
+    // When initiating a rollback automatically as part of deployment failure handling, only rollback to a server
+    // group that's enabled, as any disabled ones, even if newer, were likely manually marked so for being "bad"
+    // (e.g. as part of a manual rollback).
+    additionalRollbackContext.disregardDisabledCandidates = true
```

Comment:
nit: Personally, I prefer something like this:
```suggestion
    additionalRollbackContext.onlyEnabledServerGroups = true
```

(or `excludeDisabledServerGroups`)

Reply:
findImageFromCluster already uses `onlyEnabled` but maybe that's too ambiguous in this context

Reply:
I went with "disabled" because that's what clouddriver uses in the response:
https://github.com/spinnaker/orca/blob/688bd3d1af8d366f425059c9be89839ffe6a71f3/orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/cluster/DetermineRollbackCandidatesTask.java#L184

Will change to `excludeDisabledServerGroups`

Reply:
Never mind, will take your suggestion.